### ISSUE TITLE
Belly slider won't go past default limit?

### ISSUE THREAD

**Issue by celestialVulpine** [portalsam1/AtlyssCharacterFreedom#3]
After a fresh install, the Belly slider is seemingly still at default, but if i set the minimum value past what the default maximum is, it'll get bigger, but then the slider becomes broken and I can't slide it. Same thing happens with boobs and bottom slider (however height and width are fine)

im using BenpInEx 5.4.23.2 am I just doing something wrong?

**Comment by portalsam1** [comment]
This has been also reported on discord, but I have been unfortunately unable to reproduce it on the latest version. I'm not sure what's going on here. The bug doesn't happen on my end and there's nothing seemingly wrong in the logs. I doubt you're doing something wrong but currently the cause is unknown.

**Comment by celestialVulpine** [comment]
I will say that I am a new player who got the game yesterday (Christmas), do you think Atlyss might have gotten an update that messed with something when trying to install mods with a fresh install? because everyone who had the mods installed previously still seem to work for them, but several newer players I've heard have the same issue? 
I do not mean to overstep, this is just a theory I had brewing since I made this issue. Thank you for taking your time to read these, and for making an amazing Atlyss mod!

**Comment by portalsam1** [comment]
I agree this is worth testing, and I apologize for my slowness to reply. This will have to unfortunately wait until I am not sick confined to bed but thank you very much!

**Comment by yuuking77779** [comment]
In my case, open the ATRYSS folder, open the ATRYSS_Data folder inside, open the profileCollections folder inside, open settings.json inside that folder, and change limitProportions from true to false. After rewriting it, it worked properly.

**Comment by portalsam1** [comment]
Ah yeah I forgot this was an option in the settings menu. It's required to be disabled for the mod to work correctly. I will add something to the README to explain.
